### ISSUE TITLE
Softmax or BEC loss ? which one should I use ？

### ISSUE THREAD

**Issue by robbine** [open-mmlab/mmclassification#783]
I am working on my homework of terrorist image classification.
The goal is to classifiy an image among different categories like soldier, weapon, explosion, normal-case, and etc. (normal-case image belongs to negative sample.)
As more and more images were collected, I found that an image may belong to multiple categories, e.g. a soldier holding a weapon.
I'm wondering if BCE loss is more appropriate for the task than softmax cross entropy loss.
First I tried to use the softmax cross entropy loss to solve the task, the false positive rate turned out to be very high. I have to introduce tremendous amount of diverse normal-case images  in order to lower FPR, in which case will increase training computation. 
Would you please give some advice on this task ? Thanks a lot !

**Comment by Ezra-Yu** [comment]
> As more and more images were collected, I found that an image may belong to multiple categories, e.g. a soldier holding a weapon.

So is that a `multi-label` task? What are your labels? single label or multi-label?

**Comment by robbine** [comment]
multi-label task.
labels are ['soldier', 'weapon', 'explosion', 'tank', 'cannon', 'plane'].
Is there a multi-label task config file which I can refer to ?
Do I need to add a norm-case label if BCE loss is adopted ?

**Comment by Ezra-Yu** [comment]
Sorry that there is no example for a multi-label task.

We have not done relevant experiments in this regard and can not give helpful recommendations.
My friend, who is doing a task about multi-label, said that `asymmetry focal loss` is better than `sigmoid + bce`. Here the threshold needs to be designed by yourself. You can try it.

**Comment by robbine** [comment]
Thanks for your advice !
I'll try both and keep updating with the progress.
